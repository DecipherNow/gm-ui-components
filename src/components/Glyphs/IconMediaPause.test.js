import React from "react";
import { shallow } from "enzyme";

import IconMediaPause from "./IconMediaPause";

describe("IconMediaPause", () => {
  it("matches snapshot", () => {
    const aIconMediaPause = shallow(<IconMediaPause />);
    expect(aIconMediaPause).toMatchSnapshot();
  });
});