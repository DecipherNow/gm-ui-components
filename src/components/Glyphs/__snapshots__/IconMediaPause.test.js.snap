// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IconMediaPause matches snapshot 1`] = `
<Icon
  backgroundColor="currentColor"
  backgroundOpacity={1}
  backgroundSizeRatio={1}
  borderColor="currentColor"
  borderOpacity={1}
  borderWidth={1}
  glyph=""
  glyphColor="currentColor"
  glyphName="Pause"
  glyphSizeRatio={1}
>
  <path
    d="M7.49999998,4.99999997 L9.49999999,4.99999997 C9.77614236,4.99999997 9.99999999,5.23505048 9.99999999,5.52499997 L9.99999999,18.475 C9.99999999,18.7649495 9.77614236,19 9.49999999,19 L7.49999998,19 C7.22385761,19 6.99999998,18.7649495 6.99999998,18.475 L6.99999998,5.52499997 C6.99999998,5.23505048 7.22385761,4.99999997 7.49999998,4.99999997 Z M14.5,4.99999997 L16.5,4.99999997 C16.7761424,4.99999997 17,5.23505048 17,5.52499997 L17,18.475 C17,18.7649495 16.7761424,19 16.5,19 L14.5,19 C14.2238576,19 14,18.7649495 14,18.475 L14,5.52499997 C14,5.23505048 14.2238576,4.99999997 14.5,4.99999997 Z"
    fillRule="nonzero"
    id="Pause"
  />
</Icon>
`;
